build/deploy/generate-deploy-examples.sh

```diff
@@ -0,0 +1,24 @@
+#! /usr/bin/env bash
+set -e
+
+SCRIPT_ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd -P)
+MANIFEST_FILE=../../deploy/examples/multus-validation.yaml
+SED=${SCRIPT_ROOT}/../../build/sed-in-place
+if [[ ${SKIP_GEN} = true ]]; then
+  echo "SKIP_GEN is set, skipping manifest  generation"
+  exit
+fi
+OUT_FILE="${SCRIPT_ROOT}/${MANIFEST_FILE}"
+TMP_FILE="$(mktemp)"
+
+START_MARKER='# THE BELOW HELP TEXT IS AUTO-GENERATED BY `make docs`$'
+END_MARKER='# THE ABOVE HELP TEXT IS AUTO-GENERATED BY `make docs`$'
+
+go run ${SCRIPT_ROOT}/../../cmd/rook/*.go multus validation run --help | sed -e '/^\s*$/d' | awk '{print "# "$0}' > ${TMP_FILE}
```

Comment:
ref

Reply:
@BlaineEXE  wrote:
> ref

sorry, what does that mean?